.pre-commit-config.yaml

```diff
@@ -1,38 +1,71 @@
+# Copyright (c) 2020, Xilinx
```

Comment:
New copyright header: Copyright (C) 2023, Advanced Micro Devices, Inc.

Reply:
Thanks for catching that!